pkgs/development/python-modules/poetry-dynamic-versioning/default.nix

```diff
@@ -5,6 +5,7 @@
 , jinja2
 , markupsafe
 , poetry-core
+, poetry
```

Comment:
This will be removed from `pythonPackages`: #209534

Reply:
the tests want to run poetry commands